tests/phpunit/integration/Modules/AnalyticsTest.php

```diff
@@ -458,4 +461,50 @@ protected function get_module_with_screen() {
 	protected function get_module_with_settings() {
 		return new Analytics( new Context( GOOGLESITEKIT_PLUGIN_MAIN_FILE ) );
 	}
+
+	public function test_parse_data_orderby() {
+		$analytics = new Analytics( new Context( GOOGLESITEKIT_PLUGIN_MAIN_FILE ) );
+
+		$reflected_parse_data_orderby_method = new ReflectionMethod( 'Google\Site_Kit\Modules\Analytics', 'parse_data_orderby' );
+		$reflected_parse_data_orderby_method->setAccessible( true );
+
+		// When there is no orderby in the request.
+		$result = $reflected_parse_data_orderby_method->invoke( $analytics, new Data_Request( 'GET', null, null, 'report', array() ) );
+		$this->assertTrue( is_array( $result ) );
+		$this->assertEmpty( $result );
+
+		// When a single order object is used.
+		$order  = array(
+			'fieldName' => 'views',
+			'sortOrder' => 'ASCENDING',
+		);
+		$result = $reflected_parse_data_orderby_method->invoke( $analytics, new Data_Request( 'GET', null, null, 'report', array( 'orderby' => $order ) ) );
+		$this->assertTrue( is_array( $result ) );
+		$this->assertEquals( 1, count( $result ) );
+		$this->assertTrue( $result[0] instanceof Google_Service_AnalyticsReporting_OrderBy );
+		$this->assertEquals( $order['fieldName'], $result[0]->getFieldName() );
+		$this->assertEquals( $order['sortOrder'], $result[0]->getSortOrder() );
```

Comment:
Please use the string literals for the expected values rather than referencing the `$order` variable. They should be the same, but this makes the test easier to read and less likely to pass when it shouldn't if something is wrong in the input.

Reply:
Updated.